Add German translation

Swarkin opened the pull request:
Adding German translation to GodSVG.

A couple things to note:
- I wasn't sure if i should change the word "Tag" to something else because I couldn't find a similar and short word for it in German. Problem is, "Tag" means "day" and could be confusing.
- Same applies to "Handles", couldn't find a similar word.
- Is the word SVG feminine, masculine or neutral in German? Which articles should be used? I assumed it to be masculine.
- All other translated strings should be good to go.

Please tell me whether those translations are alright (preferably by someone whose main language is German as well).

Kiisu-Master commented on translations/translation_sheet.csv:
```diff
@@ -1,54 +1,52 @@
+#remember_window_mode_settings,Remember window mode,Запомни режима на прозореца,Fenstermodus wiederherstellen
+#remember_window_mode_desc,"When this box is checked, the window can remain restored between sessions.","Когато тази кутийка е отметната, прозорецът може да остане малък между сесиите.","Wenn diese Option aktiviert ist, wird das Fenster zwischen Sitzungen wiederhergestellt."
+#language,Language,Език,Sprache
+#open,Open,Отвори,Öffnen
```
```suggestion
#invert_zoom,Invert zoom direction,Zoomrichtung umkehren
#invert_zoom_desc,Swaps zoom in and zoom out,Vertauscht Hinein- und Herauszoomen
#open,Open,Отвори,Öffnen
```

Kiisu-Master commented on translations/translation_sheet.csv:
```diff
@@ -1,54 +1,54 @@
+#remember_window_mode_settings,Remember window mode,Запомни режима на прозореца,Fenstermodus wiederherstellen
+#remember_window_mode_desc,"When this box is checked, the window can remain restored between sessions.","Когато тази кутийка е отметната, прозорецът може да остане малък между сесиите.","Wenn diese Option aktiviert ist, wird das Fenster zwischen Sitzungen wiederhergestellt."
+#language,Language,Език,Sprache
+#invert_zoom,Invert zoom direction,Zoomrichtung umkehren
+#invert_zoom_desc,Swaps zoom in and zoom out,Vertauscht Hinein- und Herauszoomen
```
sorry, forgot about the missing bg translation
```suggestion
#invert_zoom,Invert zoom direction,,Zoomrichtung umkehren
#invert_zoom_desc,Swaps zoom in and zoom out,,Vertauscht Hinein- und Herauszoomen
```

Swarkin replied:
@MewPurPur do you wanna fill out the missing translations here?

MewPurPur replied:
Обърни посоката на увеличение

Would appreciate if the description is tweaked to say "Swaps zoom in and zoom out with the mouse wheel"
(Разменя увеличението и намалянето с колелцето на мишката)

Alex2782 commented:
> * I wasn't sure if i should change the word "Tag" to something else because I couldn't find a similar and short word for it in German. Problem is, "Tag" means "day" and could be confusing.

A few days ago I also wanted to start translating into German, I'm not sure about "Tag" either.
Maybe better "Element" or "Objekt" ?


inkscape, german docs:
https://inkscape.org/de/ueber/features/

or "Knoten"? (Node like in Godot)

MewPurPur commented:
All these are valid names right now. Tags are a type of XML node called usually "element" in that context.

Swarkin commented:
I replaced "Tag" with "Element" now, fixed a small grammar mistake and changed the translation of `#invert_zoom_desc` to [what Mew requested](https://github.com/MewPurPur/GodSVG/pull/171#discussion_r1381788941).

MewPurPur commented:
I'm an Espeon, not a Mew actually. Anyway, thanks!

Alex2782 commented:
`Ungültige formatierung.` -> `Ungültige Formatierung.` the rest looks right at first glance. 👍

MewPurPur commented:
OK, we can slip this in a PR localizing tabs.
